### ISSUE TITLE
md-chips: add "md-selected-item-change" event/method and allow selection on readonly

### ISSUE THREAD

**Issue by stefan-schweiger** [angular/material#3413]
I'm really missing a convenient way of knowing when a `md-chip` was selected, like `md-autocomplete` has implemented it.

```HTML
<md-chips ng-model="vm.chips"
          md-selected-item-change="vm.chipSelected(item)">
</md-chips>
```

Also you should be able to select chips when the control is set to readonly, the only situation where you shouldn't be able to do this in my opinion is when the control is disabled.

**EDIT:** same goes for static chips, also some way of setting the selected chip would be a nice to have.

**Comment by iamryandrake** [comment]
When will this be merged in to master?

**Comment by topherfangio** [comment]
This was merged in 333984f243c1fa2a37f1c4b7030807c45bb9daa4 on Aug 14 and should be in the official 11.1 release which is already available. If it isn't working as you expect, please open a new issue and reference this one.
